package.json

```diff
@@ -79,8 +81,8 @@
     "auto": "^11.1.6",
     "chromatic": "^11.28.2",
     "concurrently": "^8.2.2",
-    "eslint": "^7.32.0",
-    "eslint-plugin-storybook": "^0.11.3",
```

Comment:
Why did you remove `eslint-plugin-storybook`? AFAIK it is compatible with flat-file config.

https://github.com/storybookjs/eslint-plugin-storybook#configuration-eslintconfigcmjs

Reply:
It's not the flat-file config compatibility that I was worried about.

I restored it. From a quick manual test, the rules of the Storybook plugin has no effects on files. Is it possible that I misconfigured it?

Reply:
Ah, it probably only affects regular CSF, and the preview file. We dont have much content there, but let's keep it anyway.